src/views/thread/index.js

```diff
@@ -246,6 +247,7 @@ class ThreadContainer extends React.Component<Props, State> {
         storedContent = toState(JSON.parse(localStorage.getItem(LS_KEY) || ''));
       } catch (err) {
         localStorage.removeItem(LS_KEY);
+        localStorage.removeItem(LS_KEY_EXPIRE);
```

Comment:
this file internally loads chatinput itself where time checking is done. we will just remove if there any error over here